### ISSUE TITLE
Problems about BertModel4Mix

### ISSUE THREAD

**Issue by melodysnoopy** [GT-SALT/LADA#2]
大佬们好，BertModel4Mix中的target_a和target_b对应的是unlabeled_data中的mask_id吧？mask_id的作用是什么呢，仅仅是作为target传到BertModel4Mix吗？mask_id的值只能取0或者1，那么意味着labels4train_a和labels4train_b在最后一个维度的0或1位置才能取到1，那么最后一个维度的size为什么是num_labels，这有什么意义呢？反正其他位置取不到1啊。这一块没怎么看懂，希望大佬解答。

**Comment by felixwzh** [comment]
Hi, thank you for your question. You are right, for unlabeled_data, the target part, i.e., the label, is not used, since we have no label info for unlabeled data. But the reason that the last dimension has the size of num_labels is that we want to have the logits, which should be of size num_labels, to compute the unsupervised loss.

**Comment by melodysnoopy** [comment]
Thanks for your timely answer, I got it.
